src/components/Loader/Loader.style.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  align-self: center;
+`;
+
+export const Dots = styled.div`
+  position: relative;
+  width: 6px;
```

Comment:
Since the next lines are identical you can export them to a function and invoke the exported function here.

example: 
```
const getDotsLayout = (delay, animation, left) => `
    left: ${left || 'auto'};
    width: 6px;
    height: 6px;
    border-radius: 5px;
    background-color: #cfcfcf;
    color: #cfcfcf;
    animation: ${animation};
    animation-delay: ${delay}s;
    `

styled.div`
${getDotsLayout(0.5, 'dotFlashing 0.7s infinite linear alternate')}

&::before {
     ${getDotsLayout(0, 'dotFlashing 0.7s infinite alternate', -10)}
}  
`
```

P.S. consider also in using the theme's util getColor for the color setup.

Reply:
Sure!